src/index.ts

```diff
@@ -0,0 +1 @@
+export {RosettaClient} from './icp_rosetta/client';
```

Comment:
EOF line missing.

Something that eslint could also take care of.

Reply:
Will the library only expose this?

Normally we also expose related types for developers to import as well.